habitat/tasks/nav/nav.py

```diff
@@ -1099,7 +1099,7 @@ def step(
         """
 
         if not isinstance(rotation, list):
-            rotation = list(rotation)
+            rotation = list(rotation)  # type: ignore
```

Comment:
This should not be ignored though. This clearly a bug in the function signature since rotation should be a List according to the typing. We may want to make the input a SizedSequence instead.

Reply:
Or alternatively a Sequence or an Iterable